cypress/support/pages/tasks/pageTasks.js

```diff
@@ -72,6 +97,47 @@ class Tasks {
     cy.get(Tasks.#visibilityDueDateInput).type(`{moveToStart}${startDueText}${visibilityDueTime}`)
   }
 
+  setTaskText(taskText){
+    cy.get(Tasks.#homeWorkDescriptionP).find('div > p').clear()
+    cy.get(Tasks.#homeWorkDescriptionP).find('div > p').type(taskText)
+  }
+
+  executeFileUpload(fileName){
+    // mark our window object to "know" when it gets reloaded
+    cy.window().then(w => w.beforeReload = true)
+    // initially the new property is there
+    cy.window().should('have.prop', 'beforeReload', true)
+    // Upload a file includes a reload of the page
+    cy.get(Tasks.#fileUploadInput)
+      .attachFile(fileName)
+    // after reload the property should be gone
+    cy.window().should('not.have.prop', 'beforeReload')
+  }
+
+  clickOnPublicSubmissionCheckbox(){
+    cy.get(Tasks.#publicSubmissionsCheckbox).click()
+  }
+
+  clickCancelInConfirmationWindow() {
```

Comment:
method name should be then --> clickOnCancelInConfirmationWindow()

Reply:
done